mentoring/README.md

```diff
@@ -6,7 +6,10 @@ The CNCF participates in and runs several mentorship programs throughout the yea
 
 ## Mission
 
-Encourage cloud native computing adoption by providing opportunities for a diverse group of new contributors to work on CNCF projects with experienced mentorship.
+The Mentorship Working group
```

Comment:
I had this wrong initially, but we should be consistent with the WG name
```suggestion
The Mentoring Working Group
```

Reply:
Updated!